src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependencyTreeTelemetryService.cs

```diff
@@ -6,8 +6,6 @@
 using System.ComponentModel.Composition;
 using System.Linq;
 using System.Threading.Tasks;
-
```

Comment:
This would seem to violate our .editorconfig `dotnet_separate_import_directive_groups` rule :P